src/component/movement/ControlledJetpack.gd

```diff
@@ -0,0 +1,31 @@
+extends Node2D
+
+export (float) var jetpack_speed = 1500
+
+
+onready var parent = get_parent()
+
+var active = false
+
+
+func _physics_process(delta):
+	var velocity = parent.velocity
+	
+	if active:
+		var target_speed = 0
+		if Input.is_action_pressed("move_up"):
+			target_speed += -1
+		if Input.is_action_pressed("move_down"):
+			target_speed +=  1
```

Comment:
Should both of these be `+1`?

Reply:
No? We're changing the `y` axis here. Up is negative, down is positive, as I'm sure you already know is the case in the programming world.

Reply:
Nah, I just didn't notice the `-1`. This would be better written as `+= 1` and `-= 1` but it doesn't really matter.